packages/utils/crates/utils-misc/src/time.rs

```diff
@@ -1,21 +1,14 @@
-
-#[cfg(not(wasm))]
-pub fn current_timestamp() -> u64 {
-    use std::time::{SystemTime, UNIX_EPOCH};
-    match SystemTime::now().duration_since(UNIX_EPOCH) {
-        Ok(d) => d.as_millis() as u64,
-        Err(_) => 1,
+pub mod native {
```

Comment:
Why should we scope non-WASM stuff into an additional submodule, when the free function itself is scoped in `time` module already ?

Reply:
So that it is possible to lazily alias it using the `use`.